### PR TITLE
RHACS: Update image version number to 4.1.3 in patch release notes - Fixes #65733

### LINKED ISSUE #65733: RHACS: Update image version number to 4.1.3 in patch release notes
### Which section(s) is the issue in?

https://github.com/openshift/openshift-docs/blob/rhacs-docs-4.1/release_notes/41-release-notes.adoc#image-versions

### What needs fixing?

Update all of the images in "current version" to 4.1.3. For example:

`registry.redhat.io/advanced-cluster-security/rhacs-main-rhel8:4.1.2
`
should be:

`registry.redhat.io/advanced-cluster-security/rhacs-main-rhel8:4.1.3
`

### PR DESCRIPTION
**Pull request by himanlabs**
Version(s):

`rhacs-docs-4.1`

[Issue](https://github.com/openshift/openshift-docs/issues/65733)

[Link to docs preview
](https://65880--docspreview.netlify.app/openshift-acs/latest/release_notes/41-release-notes#image-versions_release-notes-41)

QE review: **N/A**
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/41-release-notes.adoc [hunk 1]
```diff
@@ -454,22 +454,22 @@ This can lead to inaccuracies or inconsistencies in the reported GID. This issue
...
 | Collects runtime activity in Kubernetes or {ocp} clusters.
-a| * `registry.redhat.io/advanced-cluster-security/rhacs-collector-rhel8:4.1.2`
-* `registry.redhat.io/advanced-cluster-security/rhacs-collector-slim-rhel8:4.1.2`
+a| * `registry.redhat.io/advanced-cluster-security/rhacs-collector-rhel8:4.1.3`
+* `registry.redhat.io/advanced-cluster-security/rhacs-collector-slim-rhel8:4.1.3`
 
 | Central DB
 | Postgres instance that provides the database storage for Central.
-a| `registry.redhat.io/advanced-cluster-security/rhacs-central-db-rhel8:4.1.2`
+a| `registry.redhat.io/advanced-cluster-security/rhacs-central-db-rhel8:4.1.3
```
Comment (kcarmichael08) [inline release_notes/41-release-notes.adoc:474]:
```suggestion
a| `registry.redhat.io/advanced-cluster-security/rhacs-central-db-rhel8:4.1.3`
```

**Review by kcarmichael08 — COMMENTED** [review]
Just one small change needed to ensure correct formatting in HTML output.

**Comment by kcarmichael08** [comment]
Hi @HimanshuJ011 , if you can fix the missing ``` symbol, I can merge this. Thank you!

**Comment by HimanshuJ011** [comment]
Hello @kcarmichael08 I have fixed that HTML Formatting, Thanks for your patience.

**Comment by kcarmichael08** [comment]
That's ok, thank you for making the change! It just needs the two commits squashed into one before I can merge.

**Comment by HimanshuJ011** [comment]
Hello @kcarmichael08 , I have fixed the HTML formatting and Squash both commits too can you please check it 
 thank you 
 https://github.com/openshift/openshift-docs/compare/rhacs-docs-4.1...HimanshuJ011:openshift-docs:rhacs-docs-4.1

**Comment by kcarmichael08** [comment]
Hi, I'm sorry, we are up to version 4.1.4 as of yesterday in this version, so I am closing this PR.

**Pull request closed without merging**